concert/app/src/functions/on_disconnect/index.py

```diff
@@ -29,7 +28,7 @@ def handler(event, context):
 
     resources = table.get_item(Key=connection_key)["Item"]
 
-    for subscription_arn in resources[ATTR_CONNECTION_SNS_SUBSCRIPTION_ARNS]:
+    for subscription_arn in sorted(resources[ATTR_CONNECTION_SNS_SUBSCRIPTION_ARNS]):
```

Comment:
Just out of curiosity, why sorted here?

Reply:
Because the test stubber is annoying and expects the calls in a specific order :(